test/author_test.rb

```diff
@@ -0,0 +1,33 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/book"
+require "./lib/author"
+require "pry"
+
+class AuthorTest < MiniTest::Test
+
+  def setup
+    @charlotte_bronte = Author.new({first_name: "Charlotte", last_name: "Bronte"})
+    @jane_eyre = @charlotte_bronte.write("Jane Eyre", "October 16, 1847")
+    @villette = @charlotte_bronte.write("Villette", "1853")
```

Comment:
it would be good to call our `.write` method later in a test instead of in our setup, otherwise we can never test that our author starts off with no books